tests/test_fips.py

```diff
@@ -303,7 +307,7 @@ def test_gcrypt_binary(container_per_test: ContainerData) -> None:
 
     c.check_output(
         "zypper -n in gcc libgcrypt-devel dirmngr && "
-        "gcc -Og -g3 fips-test-gcrypt.c -Wall -Wextra -Wpedantic -lgcrypt -o fips-test-gcrypt && "
+        "gcc fips-test-gcrypt.c -Wall -Wextra -Werror -lgcrypt -o fips-test-gcrypt && "
```

Comment:
Please keep some optimisation on, otherwise `-Werror` doesn't make a ton of sense.